espanol/maquina/maquina.md

```diff
@@ -3,4 +3,4 @@ Se fijará en 1977 para evitar la continuidad del régimen tras la muerte del ú
 Así, millones de vidas serán salvadas al evitar la más grande guerra en la historia de Epsilon.
 Dante ha encargado a Garth, su principal guerrero, defender la máquina tiempo mientras es reconfigurada.
 Justo antes de ser reconfigurada el ejercito derrumba la puerta e inicia un tiroteo que termina con la muerte de Garth
-y la captura de Dante.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.